Vagrantfile.local.dist

```diff
@@ -39,3 +44,13 @@ $vm_box      = "ubuntu/trusty64"
 # Set the Salt verbosity and log level
 # $salt_log_level = "info"
 # $salt_verbose = false
+
+# Set the Salt version.
+# e.g.: "stable", "stable 2015.8", "daily", "testing", "git", "git develop",
+#       "git v2015.8", "git 8c3fadf15ec183e5ce8c63739850d543617e4357"
+# $salt_version = "stable"
+
+# Set the Salt bootstrap & install script options
+# See http://enri.se/salt-bootstrap-docs for available flags & options
+# $salt_bootstrap_options = "-P -X"
+# $salt_install_args = nil
```

Comment:
Hier issie nil maar in de Vagrantfile een lege string. Moet dat niet gelijk zijn?